tensorrt_llm/bench/benchmark/__init__.py

```
@@ -105,7 +105,7 @@ def get_llm(runtime_config: RuntimeConfig, kwargs: dict):
     """
     llm_cls = LLM
 
-    if runtime_config.backend != "tensorrt":
+    if runtime_config.backend != None:
```

Ruff (0.14.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)